package.json

```diff
@@ -60,7 +60,7 @@
     "eslint": "^8.9.0",
     "eslint-config-prettier": "^8.4.0",
     "eslint-plugin-jsx-a11y": "^6.5.1",
-    "eslint-plugin-react": "^7.28.0",
+    "eslint-plugin-react": "7.28.0",
```

Comment:
Due to [this issue](https://github.com/yannickcr/eslint-plugin-react/issues/3215), I was getting 300+ linting errors which caused of not allowing me to commit anything. That's why I fixed this one for now.